applications/connecting_applications_to_services/projecting-binding-data.adoc

```diff
@@ -0,0 +1,23 @@
+[id="projecting-binding-data"]
+= Projecting binding data
+include::modules/common-attributes.adoc[]
+include::modules/servicebinding-document-attributes.adoc[]
+:context: projecting-binding-data
+
+toc::[]
+
+This section provides information on how you can consume the binding data.
+
+== Understanding the consumption of binding data
```

Comment:
IBM SG 
`If the subject is a functional overview, begin a heading with words such as Introduction or Overview rather than
contriving a pseudo-task-oriented heading that begins with Understanding, Using, Introducing, or Learning.
Understanding text analytics
Overview of text analytics`

Reply:
```suggestion
== Consumption of binding data
```

Reply:
Thanks, @rolfedh for the suggestion. I have modified the heading.